docs/tutorial/tutorial.md

```diff
@@ -132,38 +132,60 @@ After: You should see a number in each square in the rendered output.
 
 ### An Interactive Component
 
-Let's make the Square component fill in an "X" when you click it. Try changing the button tag returned in the `render()` function of the `Square` class to:
+Let's make the Square component fill in an "X" when you click it. Try changing the button tag returned in the `render()` function of the `Square` like this:
 
-```html
-<button className="square" onClick={() => alert('click')}>
-  {/* TODO */}
-</button>
+```javascript{4-6}
+class Square extends React.Component {
+  render() {
+    return (
+      <button className="square" onClick={() => alert('click')}>
+        {/* TODO */}
+      </button>
+    );
+  }
+}
 ```
 
 This uses the new JavaScript arrow function syntax. If you click on a square now, you should get an alert in your browser.
 
+Note that we're passing a function as the `onClick` prop. Doing `onClick={alert('click')}` would alert immediately instead of when the button is clicked.
+
 React components can have state by setting `this.state` in the constructor, which should be considered private to the component. Let's store the current value of the square in state, and change it when the square is clicked. First, add a constructor to the class to initialize the state:
 
-```javascript
+```javascript{2-7}
 class Square extends React.Component {
   constructor() {
     super();
     this.state = {
       value: null,
     };
   }
-  // ...
+  render() {
+    // same as above
```

Comment:
Maybe just copy and paste it here? With highlighting it's not really that distracting, and it helps to stay in the flow.